EKS_DISTRO_MINIMAL_BASE_GIT_TAG_FILE

```diff
@@ -1 +1 @@
-2022-04-11-1649641850.2
+2022-04-27-1651020237.2
```

Comment:
I am checking this in along with these changes because the new version of libgit2 is only compatible with this new version of the source controller.